src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
Basically, these were circular dependencies which are error-prone and could've silently changed order of postsets at any time implementation is changed. Because `FS.staticInit` (which is called during `FS__postset`) needs `NODEFS` and `TTY` to be already available and initialized, it's `FS` that should be depending on the other two, and these should be initialized independently from `FS` before it's created.